main/src/test/java/org/mobilitydata/gtfsvalidator/cli/ValidateCliParametersTest.java

```diff
@@ -0,0 +1,19 @@
+package org.mobilitydata.gtfsvalidator.cli;
+
+import org.junit.Test;
+
+public class ValidateCliParametersTest {
+
+    @Test
+    public void provideUrlAndInputCliParametersShouldThrowException() {
```

Comment:
For consistency should be `{}`, but I assume this will be populated :).

Reply:
Indeed, tests are coming soon :)